README.md

```diff
@@ -15,8 +15,19 @@ Estimating the geographical range of a species from sparse observations is a cha
 # Getting Started
 
 ## :hatched_chick: Installation for local development
-Make sure you update your local branch to the latest.
+1. Clone the repository `git clone git@github.com:UMassCDS/ds4cg2024-inaturalist.git`
 
+2. Navigate to cloned project's root, run `git submodule init` and then `git submodule update`
```

Comment:
when I switched to this branch and try these commands I get the error
![image](https://github.com/user-attachments/assets/cbf01d04-d358-4207-a96c-3bee9b81d574)

probably because I still have files that were in gitignor? bot now they are not ignoring...
![image](https://github.com/user-attachments/assets/e969e6d3-07d6-421f-9db7-e649f9752aec)

Reply:
so I deleted the sinr folder and repeated the commands "git submodule init" and then "git submodule update".
now they work, but I removed my pre-trained models :)

Reply:
btw the app works as expected)

I'm not sure if it's worth adding this to the instructions, because... normal users won't have this folder to begin with)

so no action is required for this comment)

Reply:
yes I think it expects sinr to be empty

Reply:
Should we add a script that downloads and unzips pre-trained files in the submodule? so we don't have to do it ourselves each time?